docs/UserGuide.md

```diff
@@ -97,21 +103,23 @@ Adds a pet owner to PawPatrol.
 </ul>
 </div>
 
-Format: `owner i/IC_NUMBER n/NAME p/PHONE e/EMAIL a/ADDRESS`
+<div markdown="block" class="alert alert-primary">
 
-Examples:
-* `owner i/S1234567D n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25`
+**:exclamation: More on NRIC fields:**<br>
+<ul>
+<li>Due to PDPA regulations, only the last 3 digits and last letter of the NRIC will be shown in the list view.
+<ul><li>The first 4 numbers of the NRIC will be replaced with a `X` to protect the privacy of the owner.</li></ul>
+</ul>
+</div>
 
 > 2 owners may not share the same IC_NUMBER and will be deemed as duplicate
```

Comment:
Should be "deemed as duplicates". I'll update this in a later commit. Will merge in first